data/Gaussian/basicGaussian16/water_ipcm.com

```diff
@@ -0,0 +1,13 @@
+%Chk="water.chk"
```

Comment:
Could you rename all of the data files to be `water_solvent_<model>*`?

I'm also thinking it's better to have them all be Hartree-Fock, so `water_hf_solvent_` etc.

Reply:
Yeah ok that makes sense. I don't think the parsing would change for post-hf (I know turbomole has COSMO implemented for most of it's coupled cluster stuff, unsure about the other guys), but doesn't hurt to future proof